tools/code_format/check_format.py

```diff
@@ -340,6 +340,14 @@ def checkBazelTool(name, path, var):
                               "    go get -u github.com/bazelbuild/buildtools/{}".format(
                                   path, name, var, var, name, name, name))
 
+    if not BUILDIFIER_PATH or not BUILDOZER_PATH:
```

Comment:
Isn't this what `checkBazelTool` below does?

Reply:
Hi, @lizan 

When I used this tool for the first time, I didn't know I needed to install `buildifier` and `buildozer`, so I executed `tools/code_format/check_format.py fix` directly, and then It reported this error:

```
Traceback (most recent call last):
  File "tools/code_format/check_format.py", line 1118, in <module>
    ct_error_messages = format_checker.checkTools()
  File "tools/code_format/check_format.py", line 351, in checkTools
    checkBazelTool('buildifier', BUILDIFIER_PATH, 'BUILDIFIER_BIN')
  File "tools/code_format/check_format.py", line 324, in checkBazelTool
    bazel_tool_abs_path = self.lookPath(path)
  File "tools/code_format/check_format.py", line 289, in lookPath
    return shutil.which(executable) or ''
  File "/usr/local/Cellar/python@3.8/3.8.6/Frameworks/Python.framework/Versions/3.8/lib/python3.8/shutil.py", line 1365, in which
    if os.path.dirname(cmd):
  File "/usr/local/Cellar/python@3.8/3.8.6/Frameworks/Python.framework/Versions/3.8/lib/python3.8/posixpath.py", line 152, in dirname
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

So I think that showing tips will be more friendly to novices.

Reply:
Sorry for the late reply... that sounds like you should fix what checkBazelTool does, but not adding an early check before them.

Reply:
Thanks for the suggestion, i will try for it.

Reply:
I just moved the check to `checkBazelTool`, because the cause is that `BUILDIFIER_PATH` and `BUILDOZER_PATH` are `None`. What do you think? Thank you very much! @lizan